jung-io/src/main/java/edu/uci/ics/jung/io/GraphMLWriter.java

```diff
@@ -211,6 +214,11 @@ protected void writeEdgeData(Network<N, E> g, Writer w) throws IOException {
   protected void writeKeySpecification(
       String key, String type, GraphMLMetadata<?> ds, BufferedWriter bw) throws IOException {
     bw.write("<key id=\"" + key + "\" for=\"" + type + "\"");
+
+    if (null != ds.attributeName)
+      bw.write(" attr.name=\"" + ds.attributeName.replace("\"", "&quot;") + "\"");
+    if (null != ds.attributeType) bw.write(" attr.type=\"" + ds.attributeType.getValue() + "\"");
```

Comment:
It seems that these if statements are lacking braces. Please run `mvn fmt:format` to fix this and any other formatting errors I may have missed. :)

Reply:
Uhm, the formatter isn't adding the braces, I'll add them.

Reply:
Oops, very sorry, I thought it did! Thank your very much for adding them anyway. :)